app/views/spree/products/_properties.html.erb

```diff
@@ -1,13 +1,103 @@
-<% unless @product_properties.blank? %>
-  <h3 class="pt-4 font-weight-bold text-uppercase product-details-subtitle"><%= Spree.t(:details) %></h3>
+<%# unless @product_properties.blank? %>
+  <!-- <h3 class="pt-4 font-weight-bold text-uppercase product-details-subtitle"><%#= Spree.t(:details) %></h3>
   <ul id="product-properties" class="m-0 list-unstyled product-properies" data-hook>
-    <% @product_properties.each do |product_property| %>
-      <% if product_property.show_property? %>
+    <%# @product_properties.each do |product_property| %>
+      <%# if product_property.show_property? %>
         <li>
-          <span class="font-weight-bold text-break"><%= product_property.property.presentation %>:</span>
-          <span class="text-break"><%= product_property.value %></span>
+          <span class="font-weight-bold text-break"><%#= product_property.property.presentation %>:</span>
+          <span class="text-break"><%#= product_property.value %></span>
         </li>
-      <% end %>
-    <% end %>
-  </ul>
-<% end %>
+      <%# end %>
+    <%# end %>
+  </ul> -->
```

Comment:
remove this static content to new partial detail_description and add properties part here